examples/sail/default.sail

```diff
@@ -0,0 +1,7 @@
+$include <prelude.sail>
```

Comment:
Do you need a:
```
default Order dec
```

at the top to make it compile?

Reply:
Yes oops!

Reply:
Done